docs/Configuration.md

```diff
@@ -10,7 +10,7 @@ title: Configuration
 The `haul.config.js` file is a configuration used by `Haul`. The simplest config looks like this:
 
 ```js
-import { createWebpackConfig } from "haul";
+import { createWebpackConfig } from "@haul/preset-0.59";
```

Comment:
is the idea to have a preset per RN version?

Reply:
Generally yes. In case where `preset-0.59` would work just fine for `0.60.x` we can create a table with preset to RN version compatibility

Reply:
If you have any other idea how to handle version differences between RN, I'm open for suggestions.

Reply:
I think this is fine. A bit more maintenance needed, but the differences are usually gonna be negligible so it shouldn't be that bad. One thing I could advice is to store presets in a directory so the repo is cleaner when RN hits 70 or 80 😄: `@haul/presets/0.59`